packages/mobile/src/walletConnect/constants.ts

```diff
@@ -12,35 +12,23 @@ export enum SupportedActions {
   computeSharedSecret = 'personal_computeSharedSecret',
 }
 
-const actionDescriptionTranslations: { [action in SupportedActions]: string } = {
-  [SupportedActions.eth_accounts]: i18n.t('description.accounts'),
-  [SupportedActions.eth_signTransaction]: i18n.t('description.signTransaction'),
-  [SupportedActions.eth_sendTransaction]: i18n.t('description.sendTransaction'),
-  [SupportedActions.eth_signTypedData]: i18n.t('description.sign'),
-  [SupportedActions.eth_signTypedData_v4]: i18n.t('description.sign'),
-  [SupportedActions.eth_sign]: i18n.t('description.sign'),
-  [SupportedActions.personal_sign]: i18n.t('description.sign'),
-  [SupportedActions.personal_decrypt]: i18n.t('description.decrypt'),
-  [SupportedActions.computeSharedSecret]: i18n.t('description.computeSharedSecret'),
-}
-
-const actionTranslations: { [x in SupportedActions]: string } = {
-  [SupportedActions.eth_accounts]: i18n.t('action.accounts'),
-  [SupportedActions.eth_signTransaction]: i18n.t('action.signTransaction'),
-  [SupportedActions.eth_sendTransaction]: i18n.t('action.sendTransaction'),
-  [SupportedActions.eth_signTypedData]: i18n.t('action.sign'),
-  [SupportedActions.eth_signTypedData_v4]: i18n.t('action.sign'),
-  [SupportedActions.eth_sign]: i18n.t('action.sign'),
-  [SupportedActions.personal_sign]: i18n.t('action.sign'),
-  [SupportedActions.personal_decrypt]: i18n.t('action.decrypt'),
-  [SupportedActions.computeSharedSecret]: i18n.t('action.computeSharedSecret'),
-}
-
 export function isSupportedAction(action: string) {
-  return action in actionTranslations
+  return Object.values(SupportedActions).includes(action as SupportedActions)
```

Comment:
@jeanregisser this change i'm iffy about, what do you think

Reply:
Looks good! 
Nothing to do now, we could have skipped the enum and used `type SupportedActions = "eth_accounts" | ...`